tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -116,12 +116,12 @@ def __init__(self,
         reserved_memory = max(1536, int(0.07 * node_memory) + 1)
         effective_node_memory = node_memory - reserved_memory
 
-        # Aim to use about half the disk space: set retention limits
-        # to enforce that.  This enables traffic tests to run as long
-        # as they like without risking filling the disk.
         partition_replicas_per_node = int(
             (self.partition_limit * replication_factor) / node_count)
 
+        # Aim to use about half the disk space: set retention limits
+        # to enforce that.  This enables traffic tests to run as long
+        # as they like without risking filling the disk.
```

Comment:
Please make changes other than the one indicated by commit message a separate commit. This way we can keep Redpanda repository bisectable. We try to make sure that a single commit includes only one changes. The typo changes or comments cleanup should be separate commit in this case.